rust-support/chatdbg_macros/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{parse, ItemFn};
+
+#[proc_macro_attribute]
+pub fn main(_attr: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
One small thing is that because `_attr` is ignored, ChatDBG will silently allow anything to appear inside `#[chatdbg::main(...)]`. E.g. `#[chatdbg::main(123)]` is accepted. I don't know if that matters or not, so let me know.

Reply:
Looks good to me. I will test shortly.